core-ui/src/components/Preferences/ThemeChooser.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useTheme, Dropdown } from 'react-shared';
+import { LayoutPanel } from 'fundamental-react';
+
+const THEMES = [
+  {
+    key: 'default',
+    text: 'Light (default)',
+  },
+  // {
+  //   key: 'light_dark',
+  //   text: 'Light Dark',
+  // },
+  // I cannot see any difference from "dark"
+  {
+    key: 'dark',
+    text: 'Dark',
+  },
+  {
+    key: 'hcw',
+    text: 'High Contrast White',
+  },
+  {
+    key: 'hcb',
+    text: 'High Contrast Black',
+  },
+];
+
+export default function ThemeChooser() {
+  const { theme, setTheme } = useTheme();
+
+  return (
+    <LayoutPanel className="fd-margin--tiny fd-margin-top--md">
+      <LayoutPanel.Header>
+        <LayoutPanel.Head title="Theme" />
+        <LayoutPanel.Actions>
+          {/* Choose the theme of the app */}
```

Comment:
This is a simple component so the comment is not really necessary.

Reply:
Originally it was a text displayed on the page. Should I remove it or display it to the user?

Reply:
Remove